app/javascript/components/InputName.jsx

```diff
@@ -1,12 +1,20 @@
 import React from "react";
+import { mapContextToProps } from "react-context-toolbox";
 
-const InputName = () => {
+const InputName = props => {
   return (
     <div className="form-group">
       <label htmlFor="formGroupExampleInput">Name</label>
-      <input type="text" className="form-control" id="formGroupExampleInput" />
+      <input
+        type="text"
+        className="form-control"
+        id="formGroupExampleInput"
```

Comment:
don't use id if you don't have to, because if you reuse this component you will have collision of ids